maps/nerva/datums/nerva_jobs.dm

```diff
@@ -193,13 +193,13 @@
 	outfit_type = /decl/hierarchy/outfit/job/nerva/qm
 	access = list(access_maint_tunnels, access_mailsorting, access_cargo, access_cargo_bot, access_qm, access_mining, access_mining_station,
 	access_expedition_shuttle_helm, access_expedition, access_robotics, access_research, access_teleporter,
-	access_RC_announce, access_keycard_auth, access_heads, access_eva, access_bridge, access_hydroponics, access_chapel_office, access_library, access_bar, access_kitchen, access_janitor)
+	access_RC_announce, access_keycard_auth, access_heads, access_eva, access_bridge)
```

Comment:
```suggestion
	access_RC_announce, access_keycard_auth, access_heads, access_eva, access_bridge, access_hydroponics)
```

My only suggestion is to readd hydroponics access to the QM because hydroponics is an open access area on Nerva, and this only really affects ordering crates and maybe vending machines